app/controllers/declaration/ItemsSummaryController.scala

```diff
@@ -42,6 +44,9 @@ class ItemsSummaryController @Inject()(
   def displayForm(): Action[AnyContent] = (authenticate andThen journeyType).async { implicit request =>
     cacheService
       .fetchAndGetEntry[Seq[GovernmentAgencyGoodsItem]](cacheId, itemsId)
-      .map(items => Ok(items_summary(items.getOrElse(Seq.empty))))
+      .map(items => {
+        items.map(items => items.foreach(item => Logger.info(item.toString)))
```

Comment:
Do you want to keep logging the item? This will possibly create a bit of noise on our logs.
Possibly downgrade it to `Logger.debug`

Reply:
will remove